mlir/include/mlir/Dialect/MIOpenOps/LowerMIOpenOps.h

```diff
@@ -782,9 +839,21 @@ const ArgumentFields Conv2DRewritePattern<miopen::Conv2DBwdDataOp>::fields = {
     {"KM", "MN", "KN"},
 };
 template <>
-const miopen::ConvOpType Conv2DRewritePattern<miopen::Conv2DBwdDataOp>::convOpType =
-    miopen::ConvOpType::Conv2DBwdDataOpType;
+const miopen::ConvOpType
+    Conv2DRewritePattern<miopen::Conv2DBwdDataOp>::convOpType =
+        miopen::ConvOpType::Conv2DBwdDataOpType;
+
+template <>
+const ArgumentFields Conv2DRewritePattern<miopen::Conv2DBwdWeightOp>::fields = {
+    {2, 1, 0},
+    {"MN", "KN", "KM"},
```

Comment:
@jerryyin The transformation of wrw is similar to forward/backwardData. The difference is mapping to GemmM, GemmN and GemmK here.

Reply:
I'm aware of changes here. I didn't get why are you pointing me to here?

Reply:
Never mind.

Reply:
@whchung Is it possible to open the permission such that me/jing can fold comments when we are done. Right now there is no way to `resolve` a conflict for me it seems.

Reply:
Irrelevant to the specific tagged code. Just a reminder on a previous comment:

> I'd expect test update related with lowering in mlir/test/Dialect/MIOpen as well.

You need to add the new lowering test cases and make `check-mlir` pass.